pyo3-polars/src/error.rs

```diff
@@ -24,7 +24,7 @@ impl std::convert::From<PyPolarsErr> for PyErr {
         use PyPolarsErr::*;
         match &err {
             Polars(err) => match err {
-                PolarsError::NotFound(name) => NotFoundError::new_err(name.to_string()),
```

Comment:
Why these changes?

Reply:
Because NotFound has been deleted:
PolarsError::NotFound(name) => NotFoundError::new_err(name.to_string()),
   |                              ^^^^^^^^ variant or associated item not found in `PolarsError`
and 3 new variants are found: ColumnNotFound, SchemaFieldNotFound, StructFieldNotFound.